src/app/public/modules/paging/paging.component.ts

```diff
@@ -15,28 +15,52 @@ import {
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class SkyPagingComponent implements OnChanges {
+
+  /**
+   * Specifies the page number of the currently displayed page. Page numbers start at 1 and increment.
```

Comment:
I'd change `currently displayed` to `current`.

Reply:
Done.